run-migration.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# シーシャ店舗用のマイグレーションとシードデータ実行スクリプト
+
+echo "🚀 シーシャ店舗のマイグレーションとシードデータを実行中..."
+
+cd backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)